tests/test_constructor.py

```diff
@@ -42,25 +41,41 @@ def test_set_ac_key(self):
         assert constructor._autocomplete_key == autocomplete_key
 
     def test_ac_query(self):
-        with vcr.use_cassette("fixtures/ac.cnstrc.com/query-success.yaml"):
-            constructor = ConstructorIO(**HTTP_ARGS)
+        with my_vcr.use_cassette("fixtures/ac.cnstrc.com/query-success.yaml"):
+            constructor = ConstructorIO(**HTTPS_ARGS)
             autocompletes = constructor.query(
                 query_str="a"
             )
             assert autocompletes is not None
             assert type(autocompletes) == dict
 
     def test_add(self):
-        with vcr.use_cassette("fixtures/ac.cnstrc.com/add-success.yaml"):
+        with my_vcr.use_cassette("fixtures/ac.cnstrc.com/add-success.yaml"):
             constructor = ConstructorIO(**HTTPS_ARGS)
             resp = constructor.add(
                 item_name="boinkamoinka",
                 autocomplete_section="Search Suggestions"
             )
             assert resp is True
 
+    def test_add_remove_metadata(self):
+        with my_vcr.use_cassette("fixtures/ac.cnstrc.com/add-metadata-success.yaml"):
+            constructor = ConstructorIO(**HTTPS_ARGS)
+            resp = constructor.add(
+                item_name="Metadata Example",
+                autocomplete_section="Products",
+                url="https://metadata.example",
+                metadata={ "this_key": "this_value" }
+            )
+            assert resp is True
+            resp = constructor.remove(
```

Comment:
Yeah, I remove it after adding it so that when you re-run it you don't get an error when you try to add it again.